crates/sui-framework/packages/sui-system/sources/validator.move

```diff
@@ -586,6 +586,10 @@ module sui_system::validator {
         staking_pool::pool_token_exchange_rate_at_epoch(&self.staking_pool, epoch)
     }
 
+    public fun get_deactivation_epoch_for_validator(self: &Validator):  (bool, u64) {
```

Comment:
```suggestion
    public(friend) fun get_deactivation_epoch_for_validator(self: &Validator):  (bool, u64) {
```

Reply:
I have incorporated the suggested changes